### ISSUE TITLE
Is this live call audio denoiser

### ISSUE THREAD

**Issue by BlackSpy1231** [supertetelman/local-denoiser#1]
can reduce background noise of live call @supertetelman

**Comment by supertetelman** [comment]
Yes, that's the thought. It is for live calls and love conversations.  For now it is mostly a framework for trying out different denoising plugins.
